api/raft.cc

```diff
@@ -24,7 +24,7 @@ using namespace json;
 
 void set_raft(http_context&, httpd::routes& r, sharded<service::raft_group_registry>& raft_gr) {
     r::trigger_snapshot.set(r, [&raft_gr] (std::unique_ptr<http::request> req) -> future<json_return_type> {
-        raft::group_id gid{utils::UUID{req->param["group_id"]}};
+        raft::group_id gid{utils::UUID{req->get_path_param("group_id")}};
```

Comment:
This will be fixed by a different patch that is waiting to get it.

Reply:
sorry, i failed to remove this change when rebasing this branch. reverted this change.